tests/fast-integration/monitoring/client.js

```diff
@@ -0,0 +1,58 @@
+const axios = require('axios');
+const axiosRetry = require('axios-retry');
+
+const {
+    kubectlPortForward,
+} = require("../utils");
+
+let prometheusPort = 9090;
+
+function prometheusPortForward() {
+    return kubectlPortForward("kyma-system", "prometheus-monitoring-prometheus-0", prometheusPort);
+}
+
+async function getPrometheusActiveTargets() {
+    let responseBody = await get("/api/v1/targets?state=active");
```

Comment:
I think we need to wrap the prometheus API requests in a try{}catch{} blocks in order to handle errors

Reply:
Hmm, the underlying `get` function will retry in case of an error and only fail (throw) when the max retry count exceeded. Do you think we need something additional?